app/src/androidTest/java/com/duckduckgo/app/statistics/pixels/PixelNameTest.kt

```diff
@@ -24,11 +25,15 @@ class PixelNameTest {
     @Test
     fun verifyNoDuplicatePixelNames() {
         val existingNames = mutableSetOf<String>()
-        Pixel.PixelName.values().forEach {
+        AppPixelName.values().forEach {
+            if (!existingNames.add(it.pixelName)) {
+                fail("Duplicate pixel name found: ${it.pixelName}")
```

Comment:
Maybe be more specific towards what type of pixel is missing? Something like "Duplicate app / statistics pixel name found"